include/Ausblick.tex

```diff
@@ -1,4 +1,7 @@
 \chapter{Ausblick}
 \label{ch:Ausblick}
 
-\section{Projekt Fazit}
\ No newline at end of file
+Die im Projekt erarbeitete Machbarkeitsstudie ist qualitativ hochwertig entstanden. Sie beschreibt wichtige Erkenntnisse die im Entstehungssprozess gewonnen wurden, und für das Folgeprojekt einen informativen Mehrwert bringt. In diesem Folgeprojekt soll ein Produkt erarbeitet werden, welches sich nahtlos in den Arbeitsprozess der Speicherbibliothek eingliedert.
```

Comment:
Die im Projekt erarbeitete Machbarkeitsstudie ist qualitativ hochwertig entstanden.
_Macht für mich keinen Sinn, dass diese qualitativ hochwertig entstanden ist. Sie ist wohl ehre qualitativ hochwertig vorhanden und wurde in einer sehr guten und intensiven Teamarbeit entwickelt._

Reply:
Ja voll!